src/ConsoleRunner/nunit-console/ConsoleOptions.cs

```diff
@@ -7,6 +7,7 @@
 namespace NUnit.ConsoleRunner
 {
 	using System;
+    using System.Collections.Generic;
```

Comment:
[This and other files mix tabs and spaces. Having tabs is the reason the diff looks so weird.]

Reply:
Yes... new lines would have spaces, which has been our standard for a long time. But unchanged lines still have tabs because changing the whole file would mess up the diff even more. I put a high premium for all these changes on changing as little as possible, even at the expense of consistency. If I were to eventually go to a 2.7 release, I'd probably want to clean things up a bit more, but I'm still thinking of this as a project that will be maintained for a year or two. (see README under governance repo)